frameworks/person-escort-record/manifests/property-information.yml

```diff
@@ -0,0 +1,20 @@
+name: Property information
+order: 4
+steps:
+  -
+    name: Property
+    slug: property
+    next_step:
+      -
+        question: property-including-medication
+        value: 'Yes'
+        next_step: property-details
```

Comment:
@teneightfive to check last section

Reply:
Unfortunately this doesn't play nice as is...if this field doesn't match the value it currently redirects back to the current step.

I'll see what I can do to prevent this but a temp solution might be to add the overview path as the default next step...